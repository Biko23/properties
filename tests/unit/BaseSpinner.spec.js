import { createLocalVue, mount } from '@vue/test-utils'
import Vuetify from 'vuetify'
import BaseSpinner from '@/components/BaseSpinner.vue'

describe('BaseSpinner.vue', () => {
  const localVue = createLocalVue()
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })

  const mountFunction = options => {
    return mount(BaseSpinner, {
      localVue,
      vuetify,
      ...options,
    })
  }

  it('should have a match snapshot', () => {
    const wrapper = mountFunction()
    expect(wrapper.html()).toMatchSnapshot()
  })
})

