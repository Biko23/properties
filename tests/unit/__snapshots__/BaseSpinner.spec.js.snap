// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseDialog.vue should have a match snapshot 1`] = `<div class="spin"></div>`;

exports[`BaseSpinner.vue should have a match snapshot 1`] = `<div class="spin"></div>`;
